utils/voting.py

```diff
@@ -193,12 +193,13 @@ def confirm_vote_script(
 
     # Show detailed description of prepared voting.
     if not silent:
-        human_readable_script = decode_evm_script(
-            encoded_call_script,
-            verbose=False,
-            specific_net=CHAIN_NETWORK_NAME,
-            repeat_is_error=True,
-        )
+        # human_readable_script = decode_evm_script(
+        #     encoded_call_script,
+        #     verbose=False,
+        #     specific_net=CHAIN_NETWORK_NAME,
+        #     repeat_is_error=True,
+        # )
+        human_readable_script = []
```

Comment:
ehh, needs to be fixed in the 3rd-party lib

Reply:
Fixed in https://github.com/lidofinance/AVotesParser/releases/tag/v0.5.6